Formula/s/spotifyd.rb

```diff
@@ -35,24 +26,49 @@ class Spotifyd < Formula
   depends_on "pkgconf" => :build
   depends_on "rust" => :build
   depends_on "dbus"
-  depends_on "portaudio"
+
+  on_macos do
+    depends_on "portaudio"
+  end
+
+  on_linux do
+    depends_on "alsa-lib"
+    depends_on "pulseaudio"
+  end
 
   def install
     ENV["COREAUDIO_SDK_PATH"] = MacOS.sdk_path_if_needed if OS.mac?
 
-    system "cargo", "install", "--no-default-features",
-                               "--features", "dbus_keyring,portaudio_backend",
-                               *std_cargo_args
+    # Set the same features as upstream's "default" feature set binaries
+    # https://github.com/Spotifyd/spotifyd/blob/master/.github/workflows/cd.yml
+    features = %w[dbus_mpris rodio_backend]
+    features += if OS.mac?
+      %w[portaudio_backend]
+    else
+      %w[alsa_backend pulseaudio_backend]
+    end
+
+    system "cargo", "install", "--no-default-features", "--features", features.join(","), *std_cargo_args
   end
 
   service do
-    run [opt_bin/"spotifyd", "--no-daemon", "--backend", "portaudio"]
+    run macos: [opt_bin/"spotifyd", "--no-daemon", "--backend", "portaudio"],
+        linux: [opt_bin/"spotifyd", "--no-daemon", "--backend", "pulseaudio"]
```

Comment:
Looking at switching to:
```suggestion
    run [opt_bin/"spotifyd", "--no-daemon", "--config-path", etc/"spotifyd.conf"]
```

Still deciding if what is best default for Linux. Our prior choice of Portaudio was a bit odd as it is usually not used on Linux.